### PR TITLE
Fixed dynamic nested arrays

### PR DESCRIPTION
**Pull request by AsafMah**
Fixes dynamic nested arrays, which prevented demo_db:GithubEvent to be indexed

### CODE REVIEW

File: K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs [hunk 1]
```diff
@@ -410,7 +411,11 @@ public async Task GetFieldCaps_WithDynamicColumnArray_ReturnFieldCaps()
                 .Returns((QueryData query, RequestContext context) =>
                 {
                     string response;
-                    if (query.QueryCommandText.Contains("nested_dynamic"))
+                    if (query.QueryCommandText.Contains("nested_indexer"))
```
Comment (eladiw) [inline K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs:414]: cant it be both?

Comment (AsafMah) [reply K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs:414]: We do a request for one field at a time

**Review by eladiw — APPROVED** [review]

**Comment by YannickBrombach** [comment]
lgtm

**Review by YannickBrombach — APPROVED** [review]
lgtm

**Pull request merged**